### DISCUSSION TITLE
The competition

### DISCUSSION THREAD

**Discussion by beadon** [David00/rpi-power-monitor#79] [Ideas]
Here's a pricey competitor - do you see anything here that we should be adding to this project?

https://www.zdnet.com/home-and-office/smart-home/black-friday-smart-home-deal-2022-sense-home-energy-monitors-sale/

https://sense.com/

**Comment by beadon** [comment]
Chalk 1: I spotted a feature to send an alert if there is a floating neutral -- definitely a nice safety feature out of the box.

Chalk 2: feature for a "monthly power use per device tracked". This is the set of colored circles on their samples  a visual display of the usage.  This is a copy of the same visual monthly review that Intuit's Mint.com uses to display spending / savings.

**Comment by vulture2600** [comment]
Ive heard of this over the last few years. IIRC, what I *don't* like about it is that it requires a dedicated two-pole breaker to power it, and my panel is full!
